File: FreeCodeCamp/Python/falsyBouncer.py
# https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/falsy-bouncer
# Reprak11

def bouncer(arr):
    i = 0
    while (i < len(arr)):
        if (arr[i] == False or arr[i] == None or arr[i] == "" or arr[i] == 0):
            arr.pop(i)
        else:
            i += 1
    return arr

if __name__ == "__main__":
    print(bouncer([7, "ate", "", False, 9, None, 0, "Reprak11" ]))

File: FreeCodeCamp/Javascript/falsyBouncer.js
// https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/falsy-bouncer
// Reprak11

function bouncer(arr) {
    i = 0;
    while (i < arr.length){
        if (arr[i] == false || arr[i] == 0 || arr[i] == null || arr[i] == "" || arr[i] == undefined || arr[i] == NaN){
            arr.splice(i,1);
        }
        else{
            i++;
        }
    }
    return arr;
}

// Moncake y Taynis
function bouncer2(arr) {
    return arr.filter(el => Boolean(el) !== false);
}

console.log(bouncer2([null, NaN, 1, 2, undefined]));